restapi/dataset/item.avsc

```diff
@@ -4,15 +4,10 @@
   "name": "Item",
   "doc": "Dataset item. It contains a value and its related time.",
   "fields": [
-    {"name": "value", "type": [
-      "org.radarcns.avro.restapi.sensor.HeartRate" ,
-      "org.radarcns.avro.restapi.sensor.Acceleration" ,
-      "org.radarcns.avro.restapi.sensor.Battery" ,
-      "org.radarcns.avro.restapi.sensor.BloodVolumePulse" ,
-      "org.radarcns.avro.restapi.sensor.ElectroDermalActivity" ,
-      "org.radarcns.avro.restapi.sensor.InterBeatInterval" ,
-      "org.radarcns.avro.restapi.sensor.Temperature"
-      ] } ,
-    {"name": "effectiveTimeFrame", "type": "org.radarcns.avro.restapi.header.EffectiveTimeFrame"}
+    {"name": "sample", "type": [
+      "org.radarcns.avro.restapi.data.DoubleSample" ,
```

Comment:
Since all statical values are computed using the same values, all `DescriptiveStatistic` for period `T` will have the same quality.

BTW, there was also the idea of providing more information about the data quality running some analysis from `Restructure-HDFS-topic` while restructuring. For example we may compute the distribution of missing data: let's consider:

- a sensor generating 1 sample per minutes
- a situation where we have lost 60 samples
- hour dataset covers 1 day.

If these 60 lost messages are consecutive (meaning that we have 1h of gap), the quality of the dataset should be lesser than a dataset where we have lost 3 messages per hour every 20 min.